### DISCUSSION TITLE
Cloudwatch: Add missing AWS/Kafka namespace metric HeapMemoryAfterGC

### DISCUSSION THREAD

**Discussion by ran-ka** [grafana/grafana#66359] [Data Sources]
**What happened:**
One of the most important metrics when monitoring Kafka is memory usage. 

[AWS MSK Best practices: Monitor Apache Kafka memory](https://docs.aws.amazon.com/msk/latest/developerguide/bestpractices.html
)

We recommend that you monitor the memory that Apache Kafka uses. Otherwise, the cluster may become unavailable.

**To determine how much memory Apache Kafka uses, you can monitor the HeapMemoryAfterGC metric. HeapMemoryAfterGC is the percentage of total heap memory that is in use after garbage collection. We recommend that you create a CloudWatch alarm that takes action when HeapMemoryAfterGC increases above 60%.**

However, looking here:
https://github.com/grafana/grafana/blob/main/pkg/tsdb/cloudwatch/constants/metrics.go

I couldn't find HeapMemoryAfterGC which is the best indicator for a memory issue.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82716. Feel free to continue the discussion around this there. Thank you!
